cmd/pool/order.go

```diff
@@ -688,6 +751,20 @@ func parseBaseBid(ctx *cli.Context) (*poolrpc.Bid, *sidecar.Ticket, error) {
 		}
 	}
 
+	// Checks for the outbound liquidity market.
+	if bid.Details.AuctionType == auctioneerrpc.AuctionType_AUCTION_TYPE_BTC_OUTBOUND_LIQUIDITY {
+		baseSupply := uint64(order.BaseSupplyUnit)
+		if bid.Details.Amt != baseSupply ||
+			bid.SelfChanBalance < baseSupply {
```

Comment:
I think this should be `bid.SelfChanBalance > bid.Details.Amt`. Which I got from https://github.com/lightninglabs/pool/blob/46df668589d40a53c800c549800bc5820d73cd8d/poolrpc/trader.pb.go#L2413